comp110-worksheet-6/OxoBoard.cs

```diff
@@ -6,42 +6,168 @@
 
 namespace comp110_worksheet_6
 {
-	public enum Mark { None, O, X };
+	public enum Mark { None, O, X };   
 
-	public class OxoBoard
+    public class OxoBoard
 	{
-		// Constructor. Perform any necessary data initialisation here.
-		// Uncomment the optional parameters if attempting the stretch goal -- keep the default values to avoid breaking unit tests.
-		public OxoBoard(/* int width = 3, int height = 3, int inARow = 3 */)
+        public Mark[,] Squares;
+
+        // Constructor. Perform any necessary data initialisation here.
+        // Uncomment the optional parameters if attempting the stretch goal -- keep the default values to avoid breaking unit tests.
+        public OxoBoard(/* int width = 3, int height = 3, int inARow = 3 */)
 		{
-			throw new NotImplementedException("TODO: implement this function and then remove this exception");
+            //Sets up the boards values and sets them all to none
+            Squares = new Mark[3, 3];
+
+            for (int y = 0; y < 3; y++)
+            {
+                for (int x = 0; x < 3; x++)
+                {
+                    Squares[x, y] = Mark.None;
+                }
+            }
 		}
 
 		// Return the contents of the specified square.
 		public Mark GetSquare(int x, int y)
 		{
-			throw new NotImplementedException("TODO: implement this function and then remove this exception");
+            Mark value = Squares[x, y];
+            return value;
```

Comment:
The `value` variable isn't really needed here -- you could just `return Squares[x, y]`